learners/starter-pack.md

```diff
@@ -1,7 +1,20 @@
----
-nav_order: 0
-title: Starter Pack
-parent: For Learners
----
-
-# Starter Pack
+# Context
+
+New starters in computing roles at STFC join with a broad range of prior experience, and are generally faced with learning many new tools, programming languages and methodologies from day one. To aid with this, by providing a basic level of knowledge in some of the most widely-used computing technologies here at STFC, a series of Software Carpentry courses will be delivered later this year (see below).
+These hands-on workshops will cover basic concepts and tools including version control with Git, task automation with the Unix shell, and software development techniques employed in a scientific research-oriented environment using Python.
+
+Some new starters will start well in advance of these courses, and will soon require the basic skills covered within them; therefore, links are provided below for the course material, produced by [The Carpentries](https://carpentries.org/), which can be self-taught if desired. We also provide some additional links for further reading, covering a few techniques and tools that are outside the scope of these courses.
+
+# Workshop information

```

Comment:
It'd be good to have a 'Which courses are right for me?'-type section somewhere, since we established some folks were confused about this last year. Do you think that would be best included in the starter pack or somewhere else?

Reply:
That sounds like a good idea - what do you think about having that content somewhere else, but potentially linking to it from this page?

Reply:
Sure, that should be fine. Could you write something up for that? You can probably copy stuff from the registration form and/or the course pages.

Reply:
I'll move this to an issue as I'm happy to merge this otherwise.